packages/elements/src/calendar/__test__/calendar.defaults.test.js

```diff
@@ -147,4 +164,79 @@ describe('calendar/Defaults', function () {
       await expect(el).shadowDom.to.equalSnapshot();
     });
   });
+
+  describe('before-cell-render event fires correctly', function () {
+    it('before-cell-render event fires on first render', async function () {
+      const el = fixtureSync('<ef-calendar></ef-calendar>');
```

Comment:
Why is this test case using `fixtureSync` ? Do we have any limitation in `fixture` ?

Reply:
`fixture()` awaits for [`updateComplete`](https://lit.dev/docs/components/lifecycle/) promise to resolve first. As `before-cell-render` fires before `updateComplete` resolves, we need `fixtureSync()` to test this event.